server/utils/db/db_seed/seed.json

```diff
@@ -1,15 +1,15 @@
 {
   "users": [
-    ["Valerie", "visforvalerie@soupfanatics.com", "1234", "fbtoken1"],
-    ["Collin", "column@lamepuns.com", "asdf", "fbtoken2"],
-    ["Alissa", "a-dawg@thenuggest.com", "passwoid", "fbtoken3"],
-    ["Nate", "whiteboy@recoveringjewfros.com", "insecurepasswerd", "fbtoken4"]
+    ["Valerie", "vforvalerie@hackreactorrrr.com", "1234", "fbtoken1", "Valerie Ernst"],
```

Comment:
You appear to not have rebased before pushing these changes, madam. All of our emails have since changed.